docker/Dockerfile

```diff
@@ -14,17 +14,18 @@ ENV PYTHONPATH /app
 # Install required software.
 RUN apt-get update \
     # Enable downloading packages over https
-    && apt-get install -y apt-transport-https \
-    # Get source for node.js
-    && curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
+    && apt-get install -y apt-transport-https ca-certificates curl gnupg; \
+    # Add source for node.js
+    curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg \
```

Comment:
I'd just keep `&&` at the beginning of each line and drop `;` to make the code easier to edit.